stratio-docs/en/modules/ROOT/pages/quick-start-guide.adoc

```diff
@@ -24,7 +24,7 @@
 ** Node groups.
 ** Information required for the _Stratio KEOS_ installation.
 
-NOTE: For Kubernetes versions higher than 1.26, the installation requires a custom Ubuntu 22.04 image with necessary parameters for Elasticsearch and the ECR plugin for the _kubelet_.
+NOTE: For Kubernetes versions higher than 1.26, the installation requires a custom image with necessary parameters for Elasticsearch and the ECR plugin for the _kubelet_.
 
 ==== _control-plane_ nodes
```

Comment:
Esta caja de aviso no estaba aquí pero sí en la versión en español. La añado.
```suggestion
NOTE: In case the _cloudformation_ stack has not been previously created in the account, the `spec:security:aws:create_iam` parameter must be set to "true". Otherwise, to "false".

==== _control-plane_ nodes
```